### DISCUSSION TITLE
Support for Kimai 2.0

### DISCUSSION THREAD

**Discussion by echoview** [AlexandrePTJ/kemai#94] [General]
Hi

I really like ease which you can use Kemai to create entries in Kimai.

It makes using Kimai so simple for end users.

Question is do you have any date at this stage when hope that Kemai will support Kimai?

David

**Comment by AlexandrePTJ** [comment]
Hi,

Thanks you for using Kemai !

I am not sure to really understand your question. Kemai **does** support Kimai as it is a dedicated client. Also it works with either Kimai 1.x and 2.x.
